components/lib/screens/introPage.dart

```diff
@@ -0,0 +1,137 @@
+import 'package:flutter/material.dart';
+import 'package:intro_slider/intro_slider.dart';
+import 'package:travelsafe/screens/register.dart';
+
+class MyIntroPage extends StatefulWidget {
+  @override
+  _MyIntroPage createState() => _MyIntroPage();
+}
+
+class _MyIntroPage extends State<MyIntroPage> {
+  List<Slide> slides = new List();
+
+  @override
+  void initState() {
+    super.initState();
+
+    slides.add(
+      new Slide(
+        title: "WOMEN SAFETY",
+        styleTitle:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 30.0,
+            fontWeight: FontWeight.bold,
+            fontFamily: 'RobotoMono'),
+        description: "Travel-safe simplifies safety for women in the digital world by"
+            "making it easy to stay connected to the nearest police station.",
+        styleDescription:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 20.0,
+            fontStyle: FontStyle.italic,
+            fontFamily: 'Raleway'),
+        pathImage: "images/safety.png",
+//        pathImage: "images/women_safe.png",
+        backgroundColor: Color(0xffD02090),
+      ),
+    );
+    slides.add(
+      new Slide(
+        title: "TECHNOLOGY",
+        styleTitle:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 30.0,
+            fontWeight: FontWeight.bold,
+            fontFamily: 'RobotoMono'),
+        description: "When travelling alone, it's best to raise your guards, stay alert and make"
+            "use of the technology that entertains you and lies comfortably in your purse - your"
+            "mobile phone, for your safety.",
+        styleDescription:
+        TextStyle(color: Color(0xffFFFFFF),
+            fontSize: 20.0,
+            fontStyle: FontStyle.italic,
+            fontFamily: 'Raleway'),
+        pathImage: "images/phone.png",
```

Comment:
try with open san  for the header and roboto for content text.. if possible keep the content small. 
@tanmayrauth  we may have to decide the content